dockers/docker-teamd/supervisor-proc-exit-listener

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import os
+import signal
+import sys
+import syslog
+import swsssdk
+
+from supervisor import childutils
+
+# Contents of file should be the names of critical processes (as defined in
+# supervisor.conf file), one per line
+CRITICAL_PROCESSES_FILE = '/etc/supervisor/critical_processes'
+
+# This table in database contains the features for container and each 
+# feature in a row will be configured a state or number.
+TABLE_NAME = 'CONTAINER_FEATURE'
+
+CONTAINER_NAME = 'teamd'
+
+def main():
+    # Read the list of critical processes from a file
+    with open(CRITICAL_PROCESSES_FILE, 'r') as f:
+        critical_processes = [line.rstrip('\n') for line in f]
+
+    while True:
+        # Transition from ACKNOWLEDGED to READY
+        childutils.listener.ready()
+
+        line = sys.stdin.readline()
+        headers = childutils.get_headers(line)
+        payload = sys.stdin.read(int(headers['len']))
+
+        # Transition from READY to ACKNOWLEDGED
+        childutils.listener.ok()
+
+        # We only care about PROCESS_STATE_EXITED events
+        if headers['eventname'] == 'PROCESS_STATE_EXITED':
+            payload_headers, payload_data = childutils.eventdata(payload + '\n')
+
+            expected = int(payload_headers['expected'])
+            processname = payload_headers['processname']
+            groupname = payload_headers['groupname']
+
+            config_db = swsssdk.ConfigDBConnector()
+            config_db.connect()
+            docker_config = config_db.get_table(TABLE_NAME)
+            restart_feature = ''
```

Comment:
Suggest initialize to `None` instead of empty string.

Bonus: If you change line 50 to `restart_feature = docker_config[CONTAINER_NAME].get('auto_restart')`, it will set `restart_feature` to `None` if for some reason the key doesn't exist. This way you won't need to initialize the variable at all.

Reply:
Fixed. Good Suggestions!